### ISSUE TITLE
Is there a way to see the benchmark result in one single command

### ISSUE THREAD

**Issue by ivanceras** [krausest/js-framework-benchmark#1023]
I've been trying to run the benchmark, but it is really hard to follow the README open times there are errors when running the commands.

The README should include one command to run all the benchmarks and display the results, same as the published version of the benchmark.

It's very hard to figure out that, especially for devs that doesn't come from javascript/nodejs development.

**Comment by krausest** [comment]
Sorry, I don't see that this would be possible.
There are too many unknowns in that equation - I can't guarantee that what built yesterday builds tomorrow. Currently there are frameworks that still can't be built on a M1 mac, we've had problems on Windows with closure and angular and some packages on NPM happen to be removed such that even package-lock is lost.
The only platform that I currently build on a regular basis is the docker build: https://github.com/krausest/js-framework-benchmark/wiki/Build-all-frameworks-with-docker
